docs/authentication/social-connections/line.mdx

```diff
@@ -34,10 +34,10 @@ The next step is to create a new **LINE Login channel**. You will have to select
 
 In the next screen, after the app has been created, you will find the **Basic settings** of the application. If you want to also make email address available for your application, you will have to apply for **Email address permission** in the bottom of your app's **Basic settings**.
 
-You also need to add the **OAuth Redirect URLs.** In the Clerk Dashboard, go to **User & Authentication > [Social Connections](https://dashboard.clerk.com/last-active?path=user-authentication/social-connections)** and enable LINE. In the modal that opened, ensure **Use custom credentials** is enabled and copy **Authorized redirect URI**. Paste the value into the **LINE login → Callback URL** input.
+Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=user-authentication/sso-connections). In the top navigation, select **Configure**. Then in the sidebar, select **SSO Connections**. Select the **Add connection** button, and select **For all users**. In the **Choose provider** dropdown, select **LINE**. Toggle on **Use custom credentials** and copy **Authorized redirect URI**. Paste the value into the **LINE login → Callback URL** input.
 
 Once all the above are complete, copy the **Channel ID** and **Channel secret.** Go back to the Clerk Dashboard and paste them into the respective fields.
 
 ![Pasting value into LINE dashboard](/docs/images/authentication-providers/line/pasting-value-into-line-dashboard.jpg)
 
-Don't forget to click **Apply** in the Clerk dashboard. Congratulations! Social connection with LINE is now configured for your instance.
+Finally, click **Add connection** so that the settings are applied. Congratulations! Social connection with LINE is now configured for your instance.
```

Comment:
```suggestion
Finally, select **Add connection** so that the settings are applied. Congratulations! Social connection with LINE is now configured for your instance.
```